### DISCUSSION TITLE
Bails target user

### DISCUSSION THREAD

**Discussion by docalb** [BenWestgate/Bails#48] [General]
Who is Bails target user? Twitter anon bitcoiner who knows what bitcoin core is? Non-online family relative who wants a private way to send and store btc?

I'm wondering about UX. Assuming Bails will have it's own url at some point with cleaned up guide. If targeting unexperienced users: let them know 101-level stuff up front (syncing to blockchain takes xx hours, etc.)

**Comment by BenWestgate** [comment]
I had three users in my mind as I wrote Bails:

Person living in a jurisdiction that banned Bitcoin use and storage and raids those suspected of breaking that law.

Bails is meant to be an easy way to keep using & storing Bitcoin safely if it were made illegal to run or hold in their jurisdiction. The features like cloning that let you pass the USBs hand to hand help it distribute itself even if my repository was censored. 

Ironically the very existence of the project now reduces the chances of Bitcoin ever being banned because it would be a very effective circumvention even if your home or office was raided. Making the ban unprofitable for rulers.

I also made it to protect the privacy of those surveilled in their own home by nosy parents, spouses, evil maids, nosy girlfriends. Or to allow storing backups with family you want to inherit from you, who you trust not to steal from you but who would struggle to keep a secret if they learned or could learn your Bitcoin holdings. Defend against the excessively curious loved ones.

Lastly itself designed to be an easy all in one wallet and Tor  Browser for shopping on the Dark web, buying drugs, other contraband, or donating to blacklisted causes and individuals. The enhanced privacy here is much valued for 'illicit' transactions and a regular phone wallet wouldn't cut it. Likewise it would keep you safe if you were a recipient of funds for these services or donations.

So there's your 3 users: Someone living where Bitcoin is illegal to use or hold and law is enforced and needs plausible deniability that he/she doesn't use Bitcoin. Whoever has Family members or allows people in their home who are nosy and/or blabbermouths. People conducting transactions that may be considered illegal in some jurisdictions who need maximum transaction privacy.

It's meant to be easy to use so the majority of the economic power would not be deterred from using bitcoin because of the above threats.

**Reply by BenWestgate** [reply]
I don't assume any greater computer literacy than Tails does. But I do assume basic knowledge of Bitcoin. 

That perhaps this will be their first wallet after a custodial exchange or simple bip39 wallet.
